docs/HowToBuild.html

```diff
@@ -437,70 +437,76 @@ <h2 class="categoryTitle">Testing Plugins</h2>
-                        </ul>
-                    </ul>
-                    <li>Hit "OK" on all dialogs.</li>
-                    <li>You may need to restart FormIt for this to take effect.</li>
-                </ul>
-                --->
-            <p class="categoryDetailBody">Once you've started FormIt with the Environment Variable as described above, proceed based on the version of FormIt:</p>
-            <ul class="categoryDetailList">
-                <li><b>FormIt for Windows v17 and newer</b></li>
+                <li>Now copy and paste the following, then hit Enter:</li>
                 <ul>
-                    <li>Simply right-click on the plugin panel and choose "Debug", which will open Chrome (required).</li>
-                    <li>This will automatically use the port number you specified above in the Environment Variable.</li>
+                    <li><span class="categoryDetailCode"> <span>start "$env:ProgramFiles\Autodesk\FormIt\FormIt.exe"</span></span></li>
+                    <li>This will start FormIt using this Environment Variable, for this session.</li>
+                    <li>Note: this assumes FormIt is installed in the Program Files directory.</li>
                 </ul>
             </ul>
-            <p class="categoryDetailBody">Now you can select the plugin you want to debug from the list, and Chrome will take you to the debugger, where you can inspect elements and view the console, etc.</p>
-            <p class="categoryDetailBody">Learn more about <a href="https://doc.qt.io/qt-5/qtwebengine-debugging.html" target="_blank">Qt Web Engine Debugging and Profiling</a></p>
-        </div>
+        </ul>
+        <p class="categoryDetailBody">Once you've started FormIt with the Environment Variable as described above, proceed based on the version of FormIt:</p>
+        <ul class="categoryDetailList">
+            <li><b>FormIt for Windows v17 and newer</b></li>
+            <ul>
+                <li>Right-click on the plugin panel and choose "Debug", which will open your default browser.</li>
+                <li>This will automatically use the port number you specified above in the Environment Variable.</li>
+                <li>Important: copy the value from the browser's address bar and paste into a browser like Opera https://www.opera.com/</li>
+                <li>In Opera, you can now set breakpoints and debug your code like usual.</li>
+            </ul>
+        </ul>
+        <p class="categoryDetailBody">Now you can select the plugin you want to debug from the list, and Chrome will take you to the debugger, where you can inspect elements and view the console, etc.</p>
+        <p class="categoryDetailBody">Learn more about <a href="https://doc.qt.io/qt-5/qtwebengine-debugging.html" target="_blank">Qt Web Engine Debugging and Profiling</a></p>
+    </div>
+</div>
+
+<div class="categoryBlock" id="DebuggingWeb">
+    <h2 class="categoryTitle">Debugging plugins on web</h2>
```

Comment:
For consistency, can we just call this "FormIt Web" since it's within the "Debugging Plugins" section, and the Windows portion just says "Windows"?